redfish-core/lib/systems.hpp

```diff
@@ -2638,18 +2638,35 @@ inline void handleComputerSystemCollectionGet(
             BMCWEB_LOG_CRITICAL("Count wasn't found??");
             return;
         }
-        uint64_t* count = val->get_ptr<uint64_t*>();
-        if (count == nullptr)
+        nlohmann::json& countRef = *val;
```

Comment:
Why is this needed?

Reply:
this is to count the number of members in the list. like:
`"Members@odata.count": 2,`